rollup-common.js

```diff
@@ -423,6 +425,58 @@ export function litProdConfig({
           : []),
       ],
     },
+    // Node build
+    ...(nodeBuild
+      ? [
+          {
+            input: entryPoints.map((name) => `development/${name}.js`),
+            output: {
+              dir: `${outputDir}/node`,
+              format: 'esm',
+              preserveModules: true,
+              sourcemap: !CHECKSIZE,
+            },
+            external,
+            plugins: [
+              replace({
+                preventAssignment: true,
+                values: {
+                  // Setting NODE_MODE to true enables node-specific behaviors,
+                  // i.e. using globalThis instead of window, and shimming APIs
+                  // needed for Lit bootup.
+                  'const NODE_MODE = false': 'const NODE_MODE = true',
+                  // Other variables should behave like prod mode.
+                  'const DEV_MODE = true': 'const DEV_MODE = false',
+                  'const ENABLE_EXTRA_SECURITY_HOOKS = true':
+                    'const ENABLE_EXTRA_SECURITY_HOOKS = false',
+                  'const ENABLE_SHADYDOM_NOPATCH = true':
+                    'const ENABLE_SHADYDOM_NOPATCH = false',
+                },
+              }),
+              sourcemaps(),
+              // We want the Node build to be minified because:
+              //
+              // 1. It should be very slightly faster, even in Node where bytes
+              //    are not as important as in the browser.
+              //
+              // 2. It means we don't need a Node build for lit-element. There
+              //    is no Node-specific logic needed in lit-element. However,
+              //    lit-element and reactive-element must be consistently
+              //    minified or unminified together, because lit-element
+              //    references properties from reactive-element which will
+              //    otherwise have different names. The default export that
+              //    lit-element will use is minified.
```

Comment:
Interesting... so the alternative is to also create a node build for `lit-element`, if we wish all node builds to not be minified. I'm onboard with this based on your reasonings.

Reply:
Yeah. The other reason not listed was just to change fewer things at once. Since currently node users get the default build, which is minified, I figured we should just keep doing that, all else equal.